Frontend/CO.CDP.OrganisationApp/Pages/Users/UserSummary.cshtml.cs

```diff
@@ -0,0 +1,92 @@
+using CO.CDP.Organisation.WebApiClient;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.ComponentModel.DataAnnotations;
+using System.Security;
+using CO.CDP.OrganisationApp.Models;
+
+namespace CO.CDP.OrganisationApp.Pages.Users;
+
+[Authorize]
+public class UserSummaryModel(
+    IOrganisationClient organisationClient,
+    ISession session) : PageModel
+{
+    private const string ScopeAdmin = "ADMIN";
+
+    [BindProperty(SupportsGet = true)]
+    public Guid Id { get; set; }
+
+    [BindProperty]
+    public Guid? SignedInPersonId { get; set; }
+
+    [BindProperty] public ICollection<Organisation.WebApiClient.Person> Persons { get; set; } = [];
+
+    [BindProperty] public ICollection<PersonInviteModel> PersonInvites { get; set; } = [];
+
+    [BindProperty]
+    [Required(ErrorMessage = "Select yes to add another user")]
+    public bool? HasPerson { get; set; }
+
+    public async Task<IActionResult> OnGet(bool? selected)
+    {
+        var userDetails = session.Get<UserDetails>(Session.UserDetailsKey);
+
+        SignedInPersonId = userDetails?.PersonId;
+
+        try
+        {
+            Persons = await organisationClient.GetOrganisationPersonsAsync(Id);
+
+            AssertUserHasAdminScopeForOrganisation();
+
+            PersonInvites = await organisationClient.GetOrganisationPersonInvitesAsync(Id);
+        }
+        catch (ApiException ex) when (ex.StatusCode == 404)
+        {
+            return Redirect("/page-not-found");
+        }
+        catch (SecurityException se)
```

Comment:
Removed unused `se` variable 

Also when do you think `SecurityException` will be raised?

Reply:
Its thrown inside AssertUserHasAdminScopeForOrganisation()

Reply:
That's a [system defined exception](https://learn.microsoft.com/en-us/dotnet/api/system.security.securityexception?view=net-8.0) we should not be using that, in this scenario ideally we need to create our own exception.
Or in your case just return bool flag and redirect

Reply:
Ok. I'll do that